en/after-payment/status-cycle.md

```diff
@@ -39,48 +38,50 @@ It is because of customer may not yet choose any payment method within the Snap
 
 Fraud Status | 🔍 | Description
 --- | --- | ---
-`accept` | ✅ | Transaction is safe, not considered as fraud.
-`deny` | ❌ | Transaction is considered as fraud. And denied/rejected for safety reason.
-`challenge` | ⚠️ | Transaction have indication of potential fraud, but cannot be determined precisely. <br>Merchant should take action to accept or deny via Dashboard, or via [Approve](https://api-docs.midtrans.com/#approve-transaction) or [Deny](https://api-docs.midtrans.com/#deny-transaction) API
+`accept` | ✅ | Transaction is safe to proceed. It is not considered as fraud. Refer to [Accepting Fraudulent Transaction](en/after-payment/dashboard-usage.md) for more details. 
```

Comment:
To refer link to specific section of a page, you can use the following link format:

```
[Accepting Fraudulent Transaction](en/after-payment/dashboard-usage.md#challenge-transaction-accept-deny)
```
notice the `#challenge-transaction-accept-deny`